sdk/monitor/monitor-query/src/logsQueryClient.ts

```diff
@@ -56,17 +52,14 @@ export class LogsQueryClient {
    * @param options - Options for the LogsClient.
    */
   constructor(tokenCredential: TokenCredential, options?: LogsQueryClientOptions) {
-    const authPolicy = bearerTokenAuthenticationPolicy(
-      tokenCredential,
-      options?.scopes ?? defaultMonitorScope
-    );
-
     // This client defaults to using 'https://api.loganalytics.io/v1' as the
     // host.
-    const serviceClientOptions = createPipelineFromOptions(options || {}, authPolicy);
+    const serviceClientOptions = createPipelineFromOptions({ ...options });
```

Comment:
I do not think you need to call this yourself unless you know what you're doing.

Reply:
oh I am not sure - i thought this would be a way to transition code to core v2